gateway-spi/src/main/java/org/apache/knox/gateway/services/security/token/JWTokenAttributesBuilder.java

```diff
@@ -32,6 +34,9 @@ public class JWTokenAttributesBuilder {
   private boolean managed;
   private String jku;
   private String type;
+  private Set<String> groups;
+  private String kid;
+  private String issuer = "KNOXSSO";
```

Comment:
I think this was like this before, but should the issuer really be KNOXSSO in ALL cases?

Reply:
I thought we made this configurable at some point. That change would likely be better as a separate PR though.

Reply:
We have a `jwt.expected.issuer` config param which is used in `validateToken`. But the it seems the issuer used by the `issueToken` has always been `KNOXSSO`

Reply:
I agree that it need not be part of this PR, but I think it should be a configurable value.